examples/server.rs

```diff
@@ -35,42 +41,55 @@ async fn run_server() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
         Some(ref p) => p.to_owned(),
         None => "1337".to_owned(),
     };
-    let addr = format!("127.0.0.1:{}", port).parse()?;
+    let addr: SocketAddr = format!("127.0.0.1:{}", port).parse()?;
 
     // Load public certificate.
     let certs = load_certs("examples/sample.pem")?;
     // Load private key.
     let key = load_private_key("examples/sample.rsa")?;
-    // Build TLS configuration.
+
+    println!("Starting to serve on https://{}.", addr);
```

Comment:
nit: I'd drop the trailing `.` here:
```suggestion
    println!("Starting to serve on https://{}", addr);
```

Reply:
See my previous comment about removing the server example